src/Makefile

```diff
@@ -1500,90 +1507,21 @@ EXTRA_CFLAGS += -fPIC
 RTOBJS := $(sort $(foreach mod,$(obj-m),$(call TORTOBJS,$(mod))))
 
 RTDEPS := $(sort $(patsubst $(OBJDIR)/%.o,$(DEPDIR)/%.d, $(RTOBJS)))
-
```

Comment:
well actually it's not just the comments, this cratered https://github.com/machinekit/machinekit/commit/f946d49f65b9ceb0e30df1d141c9e8fbfc977924

we'll rework this over the weekend